.babelrc

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
## 🔴 Your Pull Request Title
![image](https://user-images.githubusercontent.com/93039359/231861110-1eff512f-7810-4e66-ac11-a1d7de10439f.png)

## 🟢 Descriptive Pull Request Title
![image](https://user-images.githubusercontent.com/93039359/231861758-ee1681a7-67f2-4ba4-9f21-2b39c8a7b12f.png)

- [ ] It would be very nice if you make your PR title descriptive enough to make your pull request professional and descriptive 👍